index.html

```diff
@@ -3597,5 +3597,32 @@ <h3 id="applying">
         specification via the HTML5Apps project.
       </p>
     </section>
+    <section id="developer-guidance"></section>
+    <section class="appendix">
+      <h2>
+        Developer Guidance
+      </h2>
+      <p>
+        This section provides non-normative guidance for developers of web app manifest files 
+      </p>
+      <h3>
+        Additional Examples
+      </h3>
+      <p>
+        TODO: Add additional examples
+      </p>
+      <h3>
+        Interaction With Web Crawlers
+      </h3>
+      <p>
+        Like other web resources, a web app manifest should be accessible to any web browser or web crawler.
+      </p>
+      <p>
+        If a web app developer wants to block traffic from web crawlers, 
```

Comment:
I personally think that  _`"wants to block traffic"`_ is not the correct wording here, and instead, something in the lines of _`"wants to inform"`_ should be used.

From http://www.robotstxt.org/faq/nosecurity.html: 

> .../robots.txt is not intended for access control, so don't try to use it as such. Think of it as a "No Entry" sign, not a locked door.